server/src/main/java/org/elasticsearch/index/engine/VersionConflictEngineException.java

```diff
@@ -57,4 +57,10 @@ public RestStatus status() {
     public VersionConflictEngineException(StreamInput in) throws IOException {
         super(in);
     }
+
+    @Override
+    public synchronized Throwable fillInStackTrace() {
```

Comment:
@DaveCTurner I think we do not need `synchronized` this method (unlike the super's impl).

Reply:
Ah yes, that was added by IntelliJ. Well spotted. I opened #62555.